_sass/blocks/state-pages/_arrow-box.scss

```diff
@@ -134,3 +134,11 @@ $ab-border-thick: 4px;
 #revenue-process tbody th .bar {
   display: none;
 }
+
+.revenue a[href="#revenue-oil"] {
```

Comment:
This is a quick fix. This section is slated for change when we remove the links. #wontfix